### PR TITLE
docs: Add namespace parameter to additional HTTP endpoints

### PR DESCRIPTION
**Pull request by blake**
Document the namespace parameter can be specified on HTTP Check, Connect CA leaf, and Discovery Chain API endpoints.

### CODE REVIEW

**Review by freddygv — APPROVED** [review]

File: website/content/api-docs/agent/check.mdx [hunk 1]
```diff
@@ -208,8 +227,9 @@ The table below shows this endpoint's support for
   made to both addresses, and the first successful connection attempt will
   result in a successful check.
 
-- `TTL` `(string: "")` - Specifies this is a TTL check, and the TTL endpoint
-  must be used periodically to update the state of the check.
+- `TTL` `(duration: 10s)` - Specifies this is a TTL check, and the TTL endpoint
+  must be used periodically to update the state of the check within the specified
+  duration, otherwise the check will be set to the failed state.
```
Comment (freddygv) [inline website/content/api-docs/agent/check.mdx:232]:
```suggestion
- `TTL` `(duration: 10s)` - Specifies this is a TTL check, and the TTL endpoint
  must be used periodically to update the state of the check. If the check is not
  set to passing within the specified duration, then the check will be set to the failed state.
```

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/419658.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 46b1de8467eb0b0ec2b827601219a4218c9b6bb3 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 46b1de8467eb0b0ec2b827601219a4218c9b6bb3 onto `release/1.10.x` succeeded!

**Pull request merged**